src/Toast/ToastContainer.js

```diff
@@ -10,8 +10,7 @@ import toastCSS from './toastCSS';
 
 const transition = cssTransition({
   enter: 'toastSlideIn',
-  exit: 'toastSlideOut',
-  duration: 500
+  exit: 'toastSlideOut'
```

Comment:
`duration` has been removed from `cssTransition`; `react-toastify` as of v7.0.0 supports css animations directly.